internal/cue_utils/embed_gen.go

```diff
@@ -0,0 +1,17 @@
+package cue_utils
```

Comment:
Underscores in Go packages are allowed but not conventional... Suggest renaming the directory & package to 

```suggestion
package cueutils
```

Reply:
A package-level Godoc comment might be a useful place to explain what this is and what it's for. It'd go into https://pkg.go.dev/github.com/snyk/go-application-framework

Reply:
Renamed in: https://github.com/snyk/go-application-framework/pull/236/commits/86da075ee9401380a34c0a22b036b60518b8c225

Reply:
https://github.com/snyk/go-application-framework/pull/236/commits/1769935b2307be2148d36892880f3089155195c6